src/MICore/Transports/LocalUnixTransport.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+
+namespace MICore
+{
+    public class LocalUnixTransport : StreamTransport
```

Comment:
nit: LocalUnixTerminalTransport since what is special about this is that we are using the terminal?